tests/bootstrap.php

```diff
@@ -145,3 +147,8 @@
     $loader = new SchemaLoader();
     $loader->loadInternalFile(env('FIXTURE_SCHEMA_METADATA'));
 }
+
+FactoryLocator::add(
+    'Table',
+    (new TableLocator())->allowFallbackClass(false)
```

Comment:
Why? 🙂

Reply:
I wasn't able to run `vendor/bin/phpunit tests/TestCase/Http` without this.

Reply:
Strange, it's cause various other failures since we don't have concrete classes for all the tables used in tests.

Reply:
Derp, the allowFallbackClass() part shouldn't be there.